.github/workflows/rhoai-in-kind-with-ods-ci.yaml

```
@@ -11,34 +11,45 @@ permissions: {}
 
 jobs:
   rhoai-in-kind:
+    name: >-
```

actionlint (1.7.12)

[error] context "env" is not allowed here. available contexts are "github", "inputs", "matrix", "needs", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)